internal/backend/gs/config.go

```diff
@@ -16,13 +16,19 @@ type Config struct {
 	Bucket    string
 	Prefix    string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	ConnectionsLimit uint `option:"connections_limit" help:"set a limit for the number of concurrent connections (default: 5)"`
+
+	//https://cloud.google.com/storage/docs/encryption/customer-supplied-keys#gsutil
+	EncryptKey             string      `option:"encryption_key" help:"Use custom-supplied encryption key."`
```

Comment:
Shouldn't these variables also be called `EncryptionKey` and `DecryptionKeys`?

Reply:
Renamed.